lib/logstash/inputs/http.rb

```diff
@@ -69,6 +69,9 @@ class LogStash::Inputs::Http < LogStash::Inputs::Base
   # Set the truststore password
   config :keystore_password, :validate => :password
 
+  # Set the client certificate verification method. Valid methods: none, peer, force_peer
+  config :verify_mode, :validate => :string, :default => 'none'
```

Comment:
thankfully the config dsl in logstash allows us to specify a short list of possible values, so we can validate the method setting here instead of checking `else` in line 109/110. what do you think?

suggestion:
`config :verify_mode, :validate => ['none', 'peer', 'force_peer'], :default => 'none'`